frontend/src/main/components/ChatMessage/ChatDisplay.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import { Button, Stack } from 'react-bootstrap';
+import ChatMessageDisplay from './ChatMessageDisplay';
+import { useBackend } from 'main/utils/useBackend';
+
+const ChatDisplay = () => {
+  const testId = 'ChatDisplay';
+  const refreshIntervalMilliseconds = 1000;
+  const pageSize = 10;
+
+  const [selectedPage, setSelectedPage] = useState(0);
+
+  // Stryker disable all
+  const {
+    data: page,
+    
+  } = useBackend(
+    ["/api/chat/get"],
+    {
+      method: "GET",
+      url: "/api/chat/get",
+      params: {
+        page: selectedPage,
+        size: pageSize,
+      }
+    },
+    {
+      content: [], totalPages: 0
+    },
+    { refetchInterval: refreshIntervalMilliseconds }
+  );
+
+  // Stryker restore all
+
+  const previousPageCallback = () => {
+    setSelectedPage(selectedPage - 1);
+  };
+
+  const nextPageCallback = () => {
+    setSelectedPage(selectedPage + 1);
+  };
```

Comment:
This will behave in a way you don't expect on batched updates. See this [thread](https://github.com/ucsb-cs156-m23/proj-happycows-m23-10am-4/pull/83#discussion_r1300711721) for a more detailed explanation.
```suggestion
  const previousPageCallback = () => {
    setSelectedPage((prevPage) => prevPage - 1);
  };

  const nextPageCallback = () => {
    setSelectedPage((prevPage) => prevPage + 1);
  };
```

Reply:
Thank you